src/pages/slots/SlotConfiguration.tsx

```diff
@@ -1,69 +1,237 @@
+        { userId: userId!, dayOfWeek: selectEnglishValue },
+        { skip: selectEnglishValue === "" }
+    );
+
+    const totalSlots = registeredSlots?.slots.length || 0;
+
+    const handleSelectValue = (event: React.ChangeEvent<HTMLSelectElement>) => {
+        const newEnglishValue = event.target.value;
+        const newSpanishValue = event.target.selectedOptions[0].text;
+        setSelectEnglishValue(newEnglishValue);
+        setSelectSpanishValue(newSpanishValue);
+    }
+
     const editCapacityRef = useRef<any>(null);
 
-    const handleConfirmModal = async () => {
+    const createSlotRef = useRef<any>(null);
+
+    const handleEditCapacityModal = async () => {
         const response = await editCapacityRef.current.submitForm();
         if (response) {
             setShowModal(false);
         }
     }
 
-    return (
-        <MainContainer>
-            {showModal && (
-                <Modal
-                    onClose={() => setShowModal(false)}
-                    showButtons={true}
-                    contentRef={editCapacityRef}
-                    onConfirm={handleConfirmModal}
-                >
-                    <EditSlotForm ref={editCapacityRef} />
-                </Modal>
-            )}
-            <TitleContainer>
-                <Title> MIS TURNOS</Title>
-            </TitleContainer>
+    const handleCreateSlotModal = async () => {
+        const response = await createSlotRef.current.submitForm();
+        if (response) {
+            createSlot({ dayOfWeek: selectEnglishValue, startTime: response.startTime, userId: userId! })
+                .unwrap()
+                .then(() => {
+                    setShowModal(false);
+                    toast.success("El turno ha sido registrado")
+                    setTimeout(() => {
+                        const elementId = `slot-${response.startTime.replace(':', '-')}`;
+                        const element = document.getElementById(elementId);
+                        if (element) {
+                            element.scrollIntoView({
+                                behavior: 'smooth',
+                                block: 'center'
+                            });
+                            element.classList.add('highlight');
+                            setTimeout(() => {
+                                element.classList.remove('highlight');
+                            }, 1500);
+                        }
+                    }, 100);
+                });
+        }
+    };
+
+    const SlotConfigurationSkeleton = () => {
+        return (
             <ScreenContainer>
                 <InputContainer>
                     <EditContainer>
                         <Label>Cupos disponibles</Label>
-                        <EditCapacity onClick={() => setShowModal(true)}>Editar
+                        <EditCapacity onClick={() => { setShowModal(true); setModalType(ModalType.EDIT); }}>Editar
```

Comment:
Acá se podría mejorar el onClick ya que se repite en varias partes y hacer lo siguiente:
const openModal = (type: ModalType) => {
  setModalType(type);
  setShowModal(true);
};
y en onClick:

```suggestion
 <EditCapacity onClick={() => {openModal(ModalType.EDIT); }}>
```